srcStatic/GameModules/DllModule.h

```diff
@@ -26,14 +26,16 @@ class DllModule : public GameModule
 	// The exported naming schema is different between console and ini functions
 	typedef void(*LoadModuleFunctionIni)(const char* iniSectionName);
 	typedef void(*LoadModuleFunctionConsole)();
-	typedef bool(*UnloadModuleFunction)();
+	typedef bool(*UnloadModuleFunctionIni)();
+	typedef void(*UnloadModuleFunctionConsole)();
```

Comment:
I believe these two types are reversed. The console module was the one with the `bool` return type.

Oh, and we should probably switch to `using` rather than `typedef`. They read in a more natural order.